css/trino.css

```diff
@@ -1,3 +1,7 @@
 .reveal h1, .reveal h2, .reveal h3, .reveal h4, .reveal h5, .reveal h6 {
   text-transform: none;
 }
+
+.reveal ul {
+ font-size: 24px;
```

Comment:
this at least fixes the large font of lists but it's just a bandaid, we need something better. I think ideally we would add more vertical space between bullet points as well.

Reply:
You can use things like `<br>` and `<br style="clear:both;"/>` to create entire line-breaks. For slightly larger gaps between text, you can use the `line-height` property.